openlibrary/templates/books/custom_carousel.html

```diff
@@ -1,16 +1,19 @@
 $def with(books=[], title="", url="", key="", min_books=1, load_more=None, test=False)
 
 $def render_carousel_cover(book, lazy):
+  $ fallback_cover = 'https://openlibrary.org/images/icons/avatar_book.png'
+  $ cover_host = '//covers.openlibrary.org'
```

Comment:
These are within the template; the urls are presently challenging to access from the template otherwise. I wouldn't suggest we try to couple solving this issue & having config vars accessible within templates (it's fine as a new issue if you want to make it)